testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -1,13 +1,15 @@
 from io import StringIO
 from collections import defaultdict
+from importlib import reload
```

Comment:
Why do we need `reload`? This is very unusual and I'd not rather have it unless absolutely necessary. 

It is not explicitly used so should be removed.

Reply:
The reload is for the checking of warnings.
The warning will only appear upon the import as I use reload to regenerate the warning so that it could be checked.